src/main/java/com/example/smartinfileoperation/SmartinFileOperationApplication.java

```diff
@@ -61,11 +61,12 @@ public static void main(String[] args)  {
 
```

Comment:
try (PrintStream ps = new PrintStream("output.txt")) {  // use try-with-resources .If try catch block is finished, it will be close automatically.